tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -572,6 +572,32 @@ def _test_cpu_wrmsr_restore(context):
     microvm_factory = context.custom["microvm_factory"]
     cpu_template = context.custom["cpu_template"]
 
+    cpu_template_dir = cpu_template if cpu_template else "none"
+    snapshot_artifacts_dir = (
+        Path(shared_names["snapshot_artifacts_root_dir_wrmsr"])
+        / context.kernel.base_name()
+        / cpu_template_dir
+    )
+
+    # It is possible that some X template is not supported on
+    # the instance where the snapshots were created and,
+    # snapshot is loaded on an instance where X is supported. This
+    # results in error since restore doesn't find the file to load.
+    # for e.g. lets suppose snapshot is created on Skylake and restored
```

Comment:
nit: "e.g." (exempli gratia) already means "for example", so "for" is superfluous.
s/lets/let's/